src/App.tsx

```diff
@@ -1,24 +1,46 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
 import React from 'react';
-import { UserWarning } from './UserWarning';
 
-const USER_ID = 0;
+import { USER_ID } from './utils/user';
+
+import { UserWarning } from './UserWarning';
+import { TodoList } from './components/TodoList';
+import { TodoFooter } from './components/TodoFooter/TodoFooter';
+import { TodoErrorMessage } from './components/TodoErrorMessage';
+import { UseTodosContext } from './utils/TodosContext';
+import { TodoHeader } from './components/TodoHeader';
 
 export const App: React.FC = () => {
+  const context = UseTodosContext();
+  const {
+    todos,
+    tempTodo,
+  } = context;
+  const isSomeTodoPresent = (Boolean(todos.length) || tempTodo);
+
   if (!USER_ID) {
     return <UserWarning />;
   }
 
   return (
-    <section className="section container">
-      <p className="title is-4">
-        Copy all you need from the prev task:
-        <br />
-        <a href="https://github.com/mate-academy/react_todo-app-add-and-delete#react-todo-app-add-and-delete">React Todo App - Add and Delete</a>
-      </p>
-
-      <p className="subtitle">Styles are already copied</p>
-    </section>
+    <div className="todoapp">
+      <h1 className="todoapp__title">todos</h1>
+
+      <div className="todoapp__content">
+
+        <TodoHeader />
```

Comment:
```suggestion
      <div className="todoapp__content">
        <TodoHeader />
```
Don't leave empty lines between parent and child elements